hello/lib/hello_web/templates/layout/root.html.heex

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <%= csrf_meta_tag() %>
-    <%= live_title_tag assigns[:page_title] || "Hello", suffix: " · Phoenix Framework" %>
```

Comment:
@Vretamoza why is this change needed?

Reply:
Because we declared the function title/0 in the LayoutView, so we can just call it since the layout template is the respective template of the LayoutView.

Reply:
@Vretamoza and the `live_title_tag` didn't work for this purpose as well?